ocaml/networkd/lib/network_config.ml

```diff
@@ -28,12 +28,19 @@ let config_file_path = "/var/lib/xcp/networkd.db"
 
 let temp_vlan = "xentemp"
 
-let bridge_naming_convention (device : string) =
-  if Astring.String.is_prefix ~affix:"eth" device then
+let bridge_name_of_device (device : string) =
+  if String.starts_with ~prefix:"eth" device then
     "xenbr" ^ String.sub device 3 (String.length device - 3)
   else
     "br" ^ device
 
+let bridge_naming_convention (device : string) pos_opt =
+  match pos_opt with
+  | Some index ->
+      "xenbr" ^ string_of_int index
+  | None ->
```

Comment:
I think the `None` is just equivalent case of a device whose name does not start with "eth". So the `bridge_naming_convention` would not be necessary any more.